src/interactions/pointerInteraction.ts

```diff
@@ -0,0 +1,105 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Interaction {
+  export class Pointer extends Interaction.AbstractInteraction {
+    private _mouseDispatcher: Dispatcher.Mouse;
+    private _touchDispatcher: Dispatcher.Touch;
+    private _overComponent = false;
+    private _pointerEnterCallback: (p: Point) => any;
+    private _pointerMoveCallback: (p: Point) => any;
+    private _pointerExitCallback: (p: Point) => any;
+
+    public _anchor(component: Component.AbstractComponent, hitBox: D3.Selection) {
+      super._anchor(component, hitBox);
+      this._mouseDispatcher = Dispatcher.Mouse.getDispatcher(<SVGElement> this._componentToListenTo.content().node());
+      this._mouseDispatcher.onMouseMove("Interaction.Pointer" + this.getID(), (p: Point) => this._handlePointerEvent(p));
+
+      this._touchDispatcher = Dispatcher.Touch.getDispatcher(<SVGElement> this._componentToListenTo.content().node());
+      this._touchDispatcher.onTouchStart("Interaction.Pointer" + this.getID(), (p: Point, e: TouchEvent) => this._handlePointerEvent(p));
+    }
+
+    private _handlePointerEvent(p: Point) {
+      var translatedP = this._translateToComponentSpace(p);
+      if (this._isInsideComponent(translatedP)) {
+        var wasOverComponent = this._overComponent;
+        this._overComponent = true;
+        if (!wasOverComponent && this._pointerEnterCallback) {
+          this._pointerEnterCallback(translatedP);
+        }
+        if (this._pointerMoveCallback) {
+          this._pointerMoveCallback(translatedP);
+        }
```

Comment:
So this allows for both enter and move to execute in the same action?  I think this is fine.  I'm just wondering if that's intended behavior.